File: src/main/java/com/destino/projeto_destino/model/pacote/pacoteFoto/foto/Foto.java
package com.destino.projeto_destino.model.pacote.pacoteFoto.foto;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.Setter;

@Entity
@Getter
@Setter
@Table(name = "fot_foto")
public class Foto {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "FOT_ID", nullable = false)
    private Long id;

    @Column(name = "FOT_NOME", length = 100, nullable = false)
    private String nome;

    @Column(name = "FOT_URL", nullable = false)
    private String url;

    public Foto() {
    }

    public Foto(String url, String nome) {
        this.url = url;
        this.nome = nome;
    }
}

File: src/main/kotlin/com/fatec/destino/model/pacote/pacoteFoto/foto/Foto.kt
package com.fatec.destino.model.pacote.pacoteFoto.foto

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "fot_foto")
class Foto (
    @Column(name = "FOT_NOME", length = 100, nullable = false)
    var nome: String? = null,
    
    @Column(name = "FOT_URL", nullable = false)
    var url: String? = null
) {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "FOT_ID", nullable = false)
    var id: Long? = null
}